### PR TITLE
Create traveling-for-status.md

### PR DESCRIPTION
**Pull request by monicazng**

### CODE REVIEW

File: src/working-here/traveling-for-status.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+   * Laundry charges for trips longer than five days.
+   * Expedited service fees, travel visas, and inoculation fees necessary for the destination.
+   * Travel insurance up to 100 USD or EUR per trip if you don't have any. We don't have company-wide travel insurance. There are travel insurance providers (scroll down for references) that can offer one-time rates or packages if you're planning more than one trip. Also, check with your bank, credit card issuer, or your usual insurance company since you might already have travel insurance included in place.
+
+* Non-reimbursable expenses examples:
+
+   * Travel programs, airline upgrades, fast tracks, or premium services.
+   * Airline/accommodation no-show, cancellation, or rebooking fees.
+   * Convenience products and fees for the use of the gym, spa, and other facilities.
+   * Hotel, flights, etc., room service, or in-room movies/entertainment.
```
Comment (Stefaniach) [inline src/working-here/traveling-for-status.md:44]: room service (meals) within the $75 per day should be covered imo (cc @johannesmw ?)

File: src/working-here/traveling-for-status.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+   * Mobile, roaming, or hotspot connection fees.
```
Comment (Stefaniach) [inline src/working-here/traveling-for-status.md:45]: Not sure about this one. People need to be connected to work and sometimes these expenses happen (e.g. long layovers, intercontinental trips when local SIM cards not available...)

File: src/working-here/traveling-for-status.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+## 🛩 Getting there
+
+The standard travel booking process uses [TravelPerk](/src/remote-at-status/how-to-use-travelperk.md) for flights, trains, and accommodation, including Airbnbs. You can create individual or group itineraries. TravelPerk helps find better rates, provides support for travel changes and cancellations, and you can still pay with a personal card, if you want, and expense it. 
+
+You can use your car to drive to the event or meeting location. In this case, you can expense the gas costs. 
+
+We generally discourage using rental cars unless strictly necessary, so if you're thinking about renting one, please check with someone in People Ops first.
+
+Usually, the earlier you plan your trip, the cheaper and better it's going to be. We suggest preparing at least one month ahead to have enough time to communicate your plans and stay within budget.
```
Comment (Stefaniach) [inline src/working-here/traveling-for-status.md:62]: I'd add here that we understand this is more difficult now during Covid.

File: src/working-here/traveling-for-status.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+Usually, the earlier you plan your trip, the cheaper and better it's going to be. We suggest preparing at least one month ahead to have enough time to communicate your plans and stay within budget. 
+
+
+### Commuting rates and guidelines
+
+* Sometimes it's more convenient to travel by train or bus within Europe. So keep that in mind as an option when you're looking at routes.
+* Flight rates for reference:
+   * Short-haul: $200-$500
+   * Long haul: $500-$1500
+* We usually travel in economy class fare with carry-on luggage included. You can add one checked luggage for trips longer than a week or when you're carrying additional baggage for Status.
```
Comment (Stefaniach) [inline src/working-here/traveling-for-status.md:71]: this sounds a bit extreme? if someone needs a little more luggage to feel comfortable for a business trip I think we should just pay for the checked bag, no q asked.

**Review by Stefaniach — CHANGES_REQUESTED** [review]

**Review by Terrytobin — APPROVED** [review]

**Pull request merged**